.travis.yml

```diff
@@ -29,6 +29,10 @@ script:
              travis_wait 30 npm run travis-update \
               && npm run test-staging \
               && npm run swap
+              # Tell Google to update the sitemap (https://support.google.com/webmasters/answer/183668?hl=en#addsitemap)
```

Comment:
Also this won't work.

Will update it.

Reply:
why wont it work? when I tested the command locally I received the message that it was added to the queue.

Reply:
Have a file called: `a.sh` containing:

```
#!/bin/bash

echo "1" \
    && echo "2" \
    # comment
    && echo "3" 
```

Execute it using `bash` and you will get:

```
./a.sh 
1
2
./a.sh: line 6: syntax error near unexpected token `&&'
./a.sh: line 6: `    && echo "3"'
```

Reply:
> why wont it work?

It's not valid bash syntax.